code/datums/components/uplink.dm

```diff
@@ -181,6 +181,16 @@
 							break
 					if(is_inaccessible)
 						continue
+			if(I.restricted_traits && I.discounted == FALSE)
+				if(ishuman(user))
```

Comment:
Why can't it go with rest of the requirements in if above?

Reply:
Consistency, but I changed them all for you